src/security-council-mgmt/SecurityCouncilManager.sol

```diff
@@ -248,27 +253,37 @@ contract SecurityCouncilManager is
         emit MemberAdded(_newMember, _cohort);
     }
 
+    function memberRotatedTo(address _member) internal view returns (address) {
+        if (
+            rotatedTo[_member] != address(0)
+                && !SecurityCouncilMgmtUtils.isInArray(_member, getBothCohorts())
```

Comment:
Instead of checking if it is in current cohorts, we can wipe the new member's `rotatedTo` entry whenever a new member joined. This allow us to have a clean `rotatedTo` mapping.